ee/clickhouse/queries/funnels/funnel_strict.py

```diff
@@ -16,7 +16,18 @@ def get_step_counts_query(self):
 
         max_steps = len(self._filter.entities)
 
-        partition_select = self.get_partition_cols(1, max_steps)
+        formatted_query = self.get_step_counts_without_aggregation_query()
+
+        return f"""
+            SELECT person_id, max(steps) AS steps {self._get_step_time_avgs(max_steps)} FROM (
+                {formatted_query}
+            ) GROUP BY person_id
+        """
+
+    def get_step_counts_without_aggregation_query(self):
```

Comment:
the name kinda implies the "without aggregation" part happens after `get_step_counts_query`.

Maybe I should rename these to: `get_step_counts_query`, `get_aggregated_step_counts_query`.

Hmm, but the existing aggregated version is `get_step_counts_query`. Maybe this makes the new naming more confusing 😂